specification/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,32 @@
+# Commitment Discount Type
+
+A commitment-based discount is a commitment for an amount of usage or spend throughout a specified term, in exchange for discounted unit pricing on that amount. The commitment may be based on quantities of resource units or monetary value, with various payment options and time frames.
+
+Commitment Discount Type is a provider-assigned name to identify the type of commitment-based discount applied to this record.
```

Comment:
```suggestion
Commitment Discount Type is a provider-assigned name to identify the type of commitment-based discount applied to the row.
```

Reply:
Should we use "label" and "describe" instead of "identify"? In the short description, the use of "identifier" almost sounds like it's an ID.
```suggestion
Commitment Discount Type is a provider-assigned label to describe the type of commitment-based discount applied to the row.
```